doc/api/buffer.md

```diff
@@ -404,14 +404,14 @@ to initialize a `Buffer` to zeroes.
 Example:
 
 ```js
-const buf = new Buffer(5);
+const buf = new Buffer(10);
```

Comment:
Thus I think the docs should make the behaviour clear, not the examples. Examples show only one scenario, the docs say how it will be for *all* scenarios. And frankly, I don't get the point of this example, it doesn't add anything to my understanding. The docs already said the memory wouldn't be initialized. But that's another issue, I guess.